pkg/container-runtime/installer.go

```diff
@@ -0,0 +1,42 @@
+// alibaba-inc.com Inc.
+// Copyright (c) 2004-2022 All Rights Reserved.
+//
+// @Author : huaiyou.cyz
+// @Time : 2022/8/7 10:29 PM
+// @File : installer
+//
+
+package container_runtime
+
+import "net"
+
+// 容器运行时安装器
+type Installer interface {
+	InstallOn([]net.IP) (Info, error)
+
+	UnInstallFrom([]net.IP) error
+
+	//Upgrade() (ContainerRuntimeInfo, error)
+	//Rollback() (ContainerRuntimeInfo, error)
+}
+
+func NewInstaller(conf Config) Installer {
+
+}
+
+type Config struct {
+	Type         string
+	LimitNofile  string `json:"limitNofile,omitempty" yaml:"limitNofile,omitempty"`
```

Comment:
should this field be `int` type? and  Is this field for checking OS configuration?